--- bootstrapper.py.orig	2022-07-25 14:56:38.000000000 +1000
+++ bootstrapper.py	2022-07-25 14:56:49.000000000 +1000
@@ -93,7 +93,7 @@
             )
 
         # get minio client
-        self.cos_client = minio.Minio(self.cos_endpoint.netloc, secure=self.secure, credentials=cred_provider)
+        self.cos_client = minio.Minio(self.cos_endpoint.netloc, secure=self.secure, credentials=cred_provider, region='us-east-1')
 
     @abstractmethod
     def execute(self) -> None:
